custom/icds_reports/utils/aggregation_helpers/distributed/agg_child_health.py

```diff
@@ -90,7 +90,7 @@ def staging_queries(self):
             # height_eligible calculation is to keep consistent with usage of
             # age_in_months_start & age_in_months_end in UCR
             ('height_eligible',
-                "SUM(CASE WHEN chm.age_in_months >= 6 AND chm.age_tranche NOT IN ('72') AND "
+                "SUM(CASE WHEN chm.age_in_months >= 0 AND chm.age_tranche NOT IN ('72') AND "
```

Comment:
I think you could just remove the >=0 clause. That would just make the statement short